AWS X-Ray integration documentation and cleanup (#8601)
DataDog/documentation: content/en/serverless/installation/nodejs.md

After you have configured your function following the steps above, you can view metrics, logs and traces on the [Serverless Homepage][2].

> ### Monitor Custom Serverless Metrics

Reviewer: Do you think this is the right header for this section? Maybe just `Monitor Custom Metrics` or `Monitor Custom Business Metrics` ?

> ### Monitor Custom Serverless Metrics

Author: ```suggestion
### Monitor Custom Business Metrics
```

  return response;
};
```

> Learn more about submitting custom metrics [here][3].

Reviewer: Let's make our CTAs more natural and integrated into the copy, like on this APM page: https://docs.datadoghq.com/tracing/app_analytics/?tab=java

> Learn more about submitting custom metrics [here][3].

Author: ```suggestion
[Enable custom metric submission][3] to get started.
```

### Enable the AWS X-Ray Integration

> Datadog’s integration with AWS X-Ray allows you to visualize serverless trace data, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, choose between setting up Datadog APM or the AWS X-Ray integration for your tracing needs. Learn more about configuring distributed tracing [here][4].

Reviewer: Do the APM docs talk about "trace data"? The bigger picture here is visualizing entire end-to-end requests to serverless functions

> Datadog’s integration with AWS X-Ray allows you to visualize serverless trace data, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, choose between setting up Datadog APM or the AWS X-Ray integration for your tracing needs. Learn more about configuring distributed tracing [here][4].

Author: ```suggestion
Datadog’s integration with AWS X-Ray allows you to visualize end-to-end serverless transactions, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, choose between setting up Datadog APM or the AWS X-Ray integration for your tracing needs. Learn more about configuring distributed tracing [here][4].
```
